app/src/main/java/ghcharactertracker/com/ghcharactertracker/Character.java

```diff
@@ -0,0 +1,175 @@
+package ghcharactertracker.com.ghcharactertracker;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by jesse.mailhot on 3/6/2018.
+ */
+
+public class Character implements Parcelable{
+    String playerName = "";
+    int level, maxHealth, curExp, nextLevelExp, money;
+    long id;
+    CharClass charClass;
+    Scenario currentScenario;
+
+    public Character(CharClass charClass) {
+        this.playerName = "";
+        this.charClass = charClass;
+        this.level = 0;
+        this.curExp = 0;
+        this.money = 0;
+        this.maxHealth = 0;
+        this.nextLevelExp = 0;
+
+        currentScenario = null;
+    }
+
+    public Character() {
+        this.playerName = "";
+        this.charClass = new CharClass(ClassName.Brute);
+        this.level = 0;
+        this.curExp = 0;
+        this.money = 0;
+        this.maxHealth = 0;
+        this.nextLevelExp = 0;
+
+        currentScenario = null;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    public void setLevel(int level) {
+        this.level = level;
+        this.maxHealth = this.charClass.getMaxHealth(this.level);
```

Comment:
Nothing to change here, just make sure to remember intrinsically that calling `setLevel` will also set `maxHealth` and `nextLevelExp`

Reply:
Oh wait:

S: You should call your setters instead of modifying the fields directly. if you setters change later, you'll have lingering code that is blatantly chaning the values